### PR TITLE
Adds support for Bytes to ColumnSumCheck

### PR DESCRIPTION
**Pull request by colindean**

### CODE REVIEW

File: src/main/scala/com/target/data_validator/validator/ColumnSumCheck.scala [hunk 1]
```diff
@@ -63,6 +63,7 @@ case class ColumnSumCheck(
       case LongType => evaluate(r.getLong(idx))
       case FloatType => evaluate(r.getFloat(idx))
       case DoubleType => evaluate(r.getDouble(idx))
+      case ByteType => evaluate(r.getByte(idx))
       case ut => throw new Exception(s"Unsupported type for $name found in schema: $ut")
```
Comment (samratmitra-0812) [inline src/main/scala/com/target/data_validator/validator/ColumnSumCheck.scala:67]: This behaviour of throwing exception for unsupported type is different from columnMaxCheck, where it is treated as a normal check failure. I think we should make both of them consistent.

Comment (colindean) [reply src/main/scala/com/target/data_validator/validator/ColumnSumCheck.scala:67]: I opened https://github.com/target/data-validator/issues/46 to handle that enhancement.

**Comment by samratmitra-0812** [comment]
@colindean Since I had to modify the same files to address @phpisciuneri 's review comments, I have included your changes for this PR in #43 .

**Comment by colindean** [comment]
We'll close this after #43 is merged, then. 

When you pulled in this PR's branch, did you copy-paste or use `git am` or `git cherry-pick`?

**Comment by phpisciuneri** [comment]
@samratmitra-0812 can you formally approve if you are satisfied?  See [About Pull Request Reviews](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews)

**Pull request merged**